site/public/js/server.js

```diff
@@ -286,6 +325,25 @@ function newUploadCourseMaterialsForm() {
     $('#overwrite-materials-flag').remove();
 }
 
+function newUploadBanner() {
+
+    createArray(1);
+
+    var files = [];
+
+    $('.popup-form').css('display', 'none');
+    var form = $("#upload-banner");
+
+    $('[name="existing-file-list"]', form).html('');
+    $('[name="existing-file-list"]', form).append('<b>'+JSON.stringify(files)+'</b>');
```

Comment:
=> var filesData = $('<b>').text(JSON.stringify(files));
Maybe something like this.